app/src/main/java/org/mozilla/focus/tabs/TabsSession.java

```diff
@@ -594,11 +599,38 @@ public void handleMessage(final Message msg) {
                 case MSG_FOCUS_TAB:
                     focusTab((Tab) msg.obj, msg.arg1);
                     break;
+                case MSG_ADDED_TAB:
+                    addedTab(msg);
+                    break;
                 default:
                     break;
             }
         }
 
+        private void notifyTabAdded(@NonNull final Tab tab, @Nullable final Bundle arguments) {
+            final Message msg = this.obtainMessage(MSG_ADDED_TAB);
+            final NotifierPojo pojo = new NotifierPojo();
+            pojo.tab = tab;
+            pojo.arguements = arguments;
+            msg.obj = pojo;
+            this.sendMessage(msg);
+        }
+
+        private void addedTab(Message msg) {
+            final NotifierPojo pojo = (NotifierPojo) msg.obj;
+
+            for (final TabsChromeListener l : this.chromeListeners) {
+                l.onTabAdded(pojo.tab, pojo.arguements);
+            }
+        }
+
+        private void notifyFocusTab(final Tab tab, final @TabsChromeListener.Factor int factor) {
```

Comment:
should notifyTabAdded() and notifyFocusTab() have consistent naming rule, such as notifyTabAdded() and notifyTabFocused()?